### ISSUE TITLE
Decrease the number of logits after removing categories?

### ISSUE THREAD

**Issue by realliyifei** [MadryLab/data-transfer#2]
Do you decrease the number of logits after removing categories? Say, you remove 100 categories, then the logits of ImageNet are still 1000 as the original number of labels or changed to 900? 

It seems that I cannot find the removing logits in your source code, so I assume it is the former. 

If yes, why do you consider keeping logits as the origin rather than decreasing the number, being equal to the number of remaining categories?
